mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -4297,10 +4298,7 @@ func expandIPAllocationPolicy(configured interface{}, networkingMode string, aut
 	l := configured.([]interface{})
 	if len(l) == 0 || l[0] == nil {
 		if networkingMode == "VPC_NATIVE" {
-			if autopilot {
-				return nil, nil
-			}
-			return nil, fmt.Errorf("`ip_allocation_policy` block is required for VPC_NATIVE clusters.")
+			return nil, nil
```

Comment:
Sending no ipAllocationPolicy causes the server to default to a VPC-native ipAllocationPolicy (useIpAliases=true).

I could not find documentation indicating a scenario where it would default to a routes-based policy.